### PR TITLE
Features/announcement

### PR DESCRIPTION
**Pull request by petterannerwall**

### CODE REVIEW

File: ExilenceNextBackend/API/Controllers/StatusController.cs [hunk 1]
```diff
@@ -18,5 +29,16 @@ public IActionResult GetStatus()
         {
             return Ok();
         }
+
+        [Route("announcement")]
+        [HttpPost]
+        public async Task<IActionResult> Announcement(AnouncementModel announcement)
```
Comment (viktorgullmark) [inline ExilenceNextBackend/API/Controllers/StatusController.cs:35]: We need [FromBody] here, right?

File: ExilenceNextBackend/API/Controllers/StatusController.cs [hunk 2]
```diff
@@ -18,5 +29,16 @@ public IActionResult GetStatus()
...
+        {
+            if (announcement.Password != _configuration.GetSection("Settings")["InstanceName"])
```
Comment (viktorgullmark) [inline ExilenceNextBackend/API/Controllers/StatusController.cs:37]: "InstanceName" -> "AnnouncementPassword"

**Review by viktorgullmark — CHANGES_REQUESTED** [review]

File: ExilenceNextBackend/API/Controllers/StatusController.cs (hunk 1 above)
Comment (petterannerwall) [reply ExilenceNextBackend/API/Controllers/StatusController.cs:35]: Nope, it works for me using postman

File: ExilenceNextBackend/API/Controllers/StatusController.cs (hunk 2 above)
Comment (petterannerwall) [reply ExilenceNextBackend/API/Controllers/StatusController.cs]: Whoops, done

**Review by viktorgullmark — APPROVED** [review]

**Pull request merged**